classes/Group/Group.mjs

```diff
@@ -0,0 +1,117 @@
+import dbDriver from "../../database/driver.mjs"
+const database = new dbDriver("mongo")
+
+export default class Group {
+    constructor(groupId) {
+        this._id = groupId
+    }
+
+    async #loadFromDB() {
+        return database.getById(this.groupId, "Group")
```

Comment:
`groupId `was instantiated as `this._id`

Reply:
+